.github/workflows/os-release-issues.yml

```diff
@@ -21,103 +11,59 @@ jobs:
-        id: check_if_plugin_repo_issue_exists
-        uses: actions-cool/issues-helper@v3
-        with:
-          actions: 'find-issues'
-          repo: opensearch-project/${{ matrix.entry.repo }}
-          token: ${{ steps.github_app_token.outputs.token }}
-          title-includes: '[RELEASE] Release version ${{ matrix.release_version }}'
-      - name: Replace Placeholders
-        if: steps.check_if_plugin_repo_issue_exists.outputs.issues == '[]'
+      - uses: actions/checkout@v3
+
+      - id: get-all-components
         run: |
-          # Read the file contents and replace the placeholders
-          file_path="../opensearch-build/.github/ISSUE_TEMPLATE/component_release_template.md"
-          RELEASE_VERSION="${{ matrix.release_version }}"
-          RELEASE_BRANCH=$(echo ${{ matrix.release_version }} | cut -d. -f1-2)
-          BUILD_REPO_ISSUE_OUTPUT=$(cat <<EOF
-          ${{ steps.check_if_build_repo_issue_exists.outputs.issues }}
-          EOF
-          )
-          RELEASE_ISSUE_NUMBER=$(echo $BUILD_REPO_ISSUE_OUTPUT | jq -r '.[0].number')
-          RELEASE_ISSUE="https://github.com/opensearch-project/opensearch-build/issues/${RELEASE_ISSUE_NUMBER}"
-          RELEASE_VERSION_X=$(echo "${{ matrix.release_version }}" | awk -F'.' '{print $1}').x
-          sed -e "s|{{RELEASE_VERSION}}|${RELEASE_VERSION}|g" -e "s|{{RELEASE_ISSUE}}|${RELEASE_ISSUE}|g" -e "s|{{RELEASE_BRANCH}}|${RELEASE_BRANCH}|g" -e "s|{{RELEASE_VERSION_X}}|${RELEASE_VERSION_X}|g" "$file_path" > "$file_path.tmp" && mv "$file_path.tmp" "$file_path"
-      - name: Create component release issue from file
-        if: steps.check_if_plugin_repo_issue_exists.outputs.issues == '[]'
-        uses: peter-evans/create-issue-from-file@v4
-        with:
-          title: '[RELEASE] Release version ${{ matrix.release_version }}'
-          content-filepath: ../opensearch-build/.github/ISSUE_TEMPLATE/component_release_template.md
-          labels: |
-            v${{ matrix.release_version }}
-          token: ${{ steps.github_app_token.outputs.token }}
-          repository: opensearch-project/${{ matrix.entry.repo }}
+          versions_array=${{ needs.list-manifest-versions.outputs.versions_matrix }}
+
+          # Remove brackets from the array string
+          versions_array="${versions_array:1:-1}"
+
+          # Split by commas and process each version
+          IFS=',' read -ra VERSIONS <<< "$versions_array"
+
+          # Initialize an array to store all entries
+          declare -a matrix_entries=()
+
+          for version in "${VERSIONS[@]}"; do
+            # Remove quotes and whitespace from version
+            version=$(echo $version | tr -d '"' | xargs)
+
+            # Get components for this version
+            components=$(yq eval '.components[].repository' "manifests/$version/opensearch-$version.yml" | \
+                        sed 's/.*\///;s/\.git$//' | \
+                        jq -R -s -c 'split("\n")[:-1]')
```

Comment:
Why need jq after you get the component repo name?

Reply:
Formatting the content from strings on each line into a single array of strings. 
Before jq:
```bash
$yq eval '.components[].repository' "manifests/2.19.0/opensearch-2.19.0.yml" | sed 's/.*\///;s/\.git$//'
OpenSearch
common-utils
opensearch-learning-to-rank-base
opensearch-remote-metadata-sdk
job-scheduler
security
```

After applying jq:
```bash
$yq eval '.components[].repository' "manifests/2.19.0/opensearch-2.19.0.yml" | sed 's/.*\///;s/\.git$//' | jq -R -s -c 'split("\n")[:-1]'
["OpenSearch","common-utils","opensearch-learning-to-rank-base","opensearch-remote-metadata-sdk","job-scheduler","security"]